### ISSUE TITLE
Inheritance security rules violated while overriding member

### ISSUE THREAD

**Issue by phinett** [AutoMapper/AutoMapper#8]
## Inheritance security rules violated while overriding member: 'Castle.Core.Logging.LevelFilteredLogger.InitializeLifetimeService()'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

Not sure if this is related to AutoMapper or Castle.Core though.
